libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.html

```diff
@@ -19,7 +19,16 @@
       [ngClass]="{
         'sky-control-label-required': required
       }"
-      >{{ headingText }}</span
+    >
+      <h3 *ngIf="headingLevel === 3" [class]="headingClass">
+        {{ headingText }}
+      </h3>
+      <h4 *ngIf="headingLevel === 4" [class]="headingClass">
+        {{ headingText }}
+      </h4>
+      <h5 *ngIf="headingLevel === 5" [class]="headingClass">
+        {{ headingText }}
```

Comment:
is it possible for these checks to be against the new type created? just so if we change those values we don't have to come back here and change them again

Reply:
the new type isn't an enum--its just a list of approved numbers. It would have to be changed here regardless because it would change the element we'd use